From b8a6f4a57de3f0f6e528df0b9fd30a5178c0e9af Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:28:11 -0400
Subject: [alsa] cs5535audio: Also needed on MIPS

Message-id: <140972823573-236-patchset-jkysela@redhat.com>
Patchwork-id: 90936
O-Subject: [RHEL 7.1 PATCH 235/442] ALSA: cs5535audio: Also needed on MIPS
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The CS5536 companion chipset is not only used on 32-bit x86 systems as
I originally thought, it is also used on MIPS Loongson/Lemote 2
systems. So let the snd-cs5535audio driver be built on MIPS too.

Signed-off-by: Jean Delvare <jdelvare@suse.de>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 61ef6f3e0b3bb8f6dd4d34dd29b2abfd3e682f32)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/Kconfig b/sound/pci/Kconfig
index e850e08..3e22b28 100644
--- a/sound/pci/Kconfig
+++ b/sound/pci/Kconfig
@@ -290,7 +290,7 @@ config SND_CS5530
 
 config SND_CS5535AUDIO
  tristate "CS5535/CS5536 Audio"
- depends on X86_32 || COMPILE_TEST
+ depends on X86_32 || MIPS || COMPILE_TEST
  select SND_PCM
  select SND_AC97_CODEC
  help
-- 
1.7.1